src/components/DisplayUser/mainUser.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div class="page">
+    <div class="container">
+      <div class="row" v-if="start_view">
+        <div class="col-12 col-start-view">
+          <img class="img-start-view" src="../../../public/assets/img/logo-transparent.png" alt="Logo">
+        </div>
+        <button class="btn btn-start-view" @click="callUserForm()">Pide tu turno</button>
+        <p class="p-start-view">
+          ¿Eres cliente distinguido?<br>
+          Pasa el Código de tu tarjeta por el lector para aprovechar sus beneficios
+        </p>
+      </div>
+      <div class="row" v-else>
+        <div class="col-12 col-user-form">
+          <img class="img-user-form" src="../../../public/assets/img/logo-transparent.png" alt="Logo">
+        </div>
+        <div class="col-12">
+          <span class="span-container-title">Armadora</span>
+          <input type="text" class="input-user-company" v-model="company">
+        </div>
+        <div class="col-12">
+          <span class="span-container-model">Modelo</span>
+          <input type="text" class="input-user-model" v-mode="model">
+        </div>
+        <div class="col-12">
+          <span class="span-container-year">Año</span>
+          <input type="text" class="input-user-year" v-model="year">
+        </div>
+        <div class="col-12">
+          <span class="span-container-engine">Motor</span>
+          <input type="text" class="input-" v-model="engine">
+        </div>
+        <button class="btn btn-queries" @click="callQueries()">Aceptar</button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'mainUser',
+  data: function () {
+    return {
+      // Page Variables
+      start_view: true,
+      // Apollo Call Variables
+      company: '',
+      model: '',
+      year: '',
+      engine: ''
+    }
+  },
+  methods: {
+    callUserForm: function () {
+      this.start_view = false;
+    },
+    callStartView: function () {
+      this.start_view = true;
+    },
+    callQueries: function () {
+      this.sendData();
+      this.clear();
+      this.callStartView();
+    },
+    sendData: function () {
+      // Apollo Handling
+      const {company, model, engine, year} = this
+      this.$apollo.mutate ({
+        mutation: require ('../../graphql/addTurns.gql'),
+        variables: {
+          company,
+          model,
+          engine,
+          year
+        }
+      })
+      // Apollo Handling
+    },
+    /*
+    speedCheck: function () {
```

Comment:
It seems like we are no longer using the speedCheck method; it's commented, but see if we still need it so it can be implemented or safely deleted.